solidity/contracts/bridge/Wallets.sol

```diff
@@ -159,6 +173,14 @@ library Wallets {
         emit WalletBtcBalanceRangeUpdated(minBtcBalance, maxBtcBalance);
     }
 
+    /// @notice Sets the wallet maximum age.
+    /// @param maxAge New value of the wallet maximum age
```

Comment:
We should put a dot at the end of the @param description (we do that throughout the Bridge.sol file).

Reply:
AFAIK the convention is the opposite. We should rather remove the dots in places where they occur

Reply:
Yes, the convention is the opposite.